packages/ui/client/components/explorer/ExplorerItem.vue

```diff
@@ -68,12 +69,22 @@ function toggleOpen() {
 }
 
 async function onRun(task: Task) {
+  if (state === 'todo') {
```

Comment:
state can be dynamic (you can call `ctx.skip`), so let's never assume that the `result.state` dictates if we can attempt to run it

it is just a reported state, it shouldn't block an execution attempt